portal/src/main/webapp/WEB-INF/jsp/cross_cancer_results.jsp

```diff
@@ -406,9 +406,11 @@ if (sessionError != null) {  %>
     <span class='annotation-item chang_hotspot' alt='{{changHotspotAlt}}'>
         <img width='{{hotspotsImgWidth}}' height='{{hotspotsImgHeight}}' src='{{hotspotsImgSrc}}' alt='Recurrent Hotspot Symbol'>
     </span>
+    <% if (showCivic) { %>
```

Comment:
@onursumer, for clarity: do you mean we can leave it like this so that v1.5.0 can be released, and move it to the front end in a later iteration?